k8s/namespaces/admin/traefik-private/traefik-private.yaml

```diff
@@ -0,0 +1,69 @@
+# required for apps that can't handle oauth-proxy and not public
+#  i.e. bulks-scan-processor
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: traefik-private
+  namespace: admin
+  annotations:
+    flux.weave.works/automated: "false"
+spec:
+  releaseName: traefik-private
+  chart:
+    git: https://github.com/hmcts/charts
+    ref: master
+    path: stable/traefik
+  valueFileSecrets:
+    - name: "traefik-values"
+  values:
+    imageTag: 1.7.26
```

Comment:
we should be using traefik2, make sure you copy the config from traefik-no-proxy should be basically the same but just a different IP